paramiko causing Warnings due to Blowfish deprecation in cryptography 37.0.0

cryptography released a new version 37.0.0 in which they deprecated blowfish: https://github.com/pyca/cryptography/pull/6922

We're now getting this warning output:
```
tests/test_dag_contents.py:7: in <module>
    import rail
.venv/lib/python3.9/site-packages/rail/__init__.py:33: in <module>
    from .operators.network.sftp_any_file_sensor import SFTPAnyFileSensor
.venv/lib/python3.9/site-packages/rail/operators/network/sftp_any_file_sensor.py:5: in <module>
    from airflow.providers.sftp.hooks.sftp import SFTPHook
.venv/lib/python3.9/site-packages/airflow/providers/sftp/hooks/sftp.py:24: in <module>
    import pysftp
.venv/lib/python3.9/site-packages/pysftp/__init__.py:12: in <module>
    import paramiko
.venv/lib/python3.9/site-packages/paramiko/__init__.py:22: in <module>
    from paramiko.transport import SecurityOptions, Transport
.venv/lib/python3.9/site-packages/paramiko/transport.py:132: in <module>
    class Transport(threading.Thread, ClosingContextManager):
.venv/lib/python3.9/site-packages/paramiko/transport.py:236: in Transport
    "class": algorithms.Blowfish,
.venv/lib/python3.9/site-packages/cryptography/utils.py:123: in __getattr__
    warnings.warn(obj.message, obj.warning_class, stacklevel=2)
E   cryptography.utils.CryptographyDeprecationWarning: Blowfish has been deprecated
```

During part of our build as paramiko appears to be referencing blowfish in it's transport options.

I'd be more then willing to submit a PR to remove blowfish from the paramiko transport ciphers to clear this deprecation warning.

--- a/paramiko/transport.py
+++ b/paramiko/transport.py
@@ -158,7 +158,6 @@ class Transport(threading.Thread, ClosingContextManager):
         "aes128-cbc",
         "aes192-cbc",
         "aes256-cbc",
-        "blowfish-cbc",
         "3des-cbc",
     )
     _preferred_macs = (
@@ -231,12 +230,6 @@ class Transport(threading.Thread, ClosingContextManager):
             "mode": modes.CTR,
             "block-size": 16,
             "key-size": 32,
-        },
-        "blowfish-cbc": {
-            "class": algorithms.Blowfish,
-            "mode": modes.CBC,
-            "block-size": 8,
-            "key-size": 16,
         },
         "aes128-cbc": {
             "class": algorithms.AES,

--- a/tests/test_transport.py
+++ b/tests/test_transport.py
@@ -196,9 +196,9 @@ class TransportTest(unittest.TestCase):
     def test_security_options(self):
         o = self.tc.get_security_options()
         self.assertEqual(type(o), SecurityOptions)
-        self.assertTrue(("aes256-cbc") != o.ciphers)
-        o.ciphers = ("aes256-cbc")
-        self.assertEqual(("aes256-cbc"), o.ciphers)
+        self.assertTrue(("aes256-cbc", "aes192-cbc") != o.ciphers)
+        o.ciphers = ("aes256-cbc", "aes192-cbc")
+        self.assertEqual(("aes256-cbc", "aes192-cbc"), o.ciphers)
         try:
             o.ciphers = ("aes256-cbc", "made-up-cipher")
             self.assertTrue(False)

--- a/tests/test_transport.py
+++ b/tests/test_transport.py
@@ -196,9 +196,9 @@ class TransportTest(unittest.TestCase):
     def test_security_options(self):
         o = self.tc.get_security_options()
         self.assertEqual(type(o), SecurityOptions)
-        self.assertTrue(("aes256-cbc", "blowfish-cbc") != o.ciphers)
-        o.ciphers = ("aes256-cbc", "blowfish-cbc")
-        self.assertEqual(("aes256-cbc", "blowfish-cbc"), o.ciphers)
+        self.assertTrue(("aes256-cbc") != o.ciphers)
+        o.ciphers = ("aes256-cbc")
+        self.assertEqual(("aes256-cbc"), o.ciphers)
         try:
             o.ciphers = ("aes256-cbc", "made-up-cipher")
             self.assertTrue(False)

